quart_2/seminar_7/my_phonebook/menu.py

```diff
@@ -0,0 +1,27 @@
+import csv_action as ca
+
+
+def menu():
+    print('*' * 35)
+    print('\t\tСПИСОК ДЕЙСТВИЙ:')
+    print('\t1. Показать все контакты')
+    print('\t2. Сгенерировать контакты')
+    print('\t3. Добавить контакт')
+    print('\t4. Удалить все контакты')
+    print('\t5. Закрыть программу')
+    print('*' * 35)
+    choice = int(input('\tВведите номер действия: '))
```

Comment:
Это все можно сделать, так:
```suggestion
    choice = int(input(
        f'{"*" * 35}\n'
        '\t\tСПИСОК ДЕЙСТВИЙ:\n'
        '\t1. Показать все контакты\n'
        '\t2. Сгенерировать контакты\n'
        '\t3. Добавить контакт\n'
        '\t4. Удалить все контакты\n'
        '\t5. Закрыть программу\n'
        f'{"*" * 35}\n'
        '\tВведите номер действия: '))
```
И не нужно несколько раз вызывать функцию `print`.